core-common/src/test/resources/surefire-jdk17.policy

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2014, 2022 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Yes, was copied from another file. Frankly, I am not an employee of Oracle, so not sure, that this &copy; will be correct. In other files there are Oracle copyright stand too and I modify existed &copy; but this file is new. I write here copyright with my name to avoid misunderstandings.